ansible/roles/druid-ingestion/templates/raw_sl_observation

```diff
@@ -1,197 +1,475 @@
+            },
+            {
+              "type": "string",
+              "name": "imp_project_title"
+            },
+            {
+              "type": "string",
+              "name": "imp_project_goal"
+            },
+            {
+              "type": "string",
+              "name": "imp_project_externalId"
+            },
+            {
+              "type": "string",
+              "name": "ancestorName"
+            },
+            {
+              "type": "string",
+              "name": "scoringSystem"
+            },
+            {
+              "type": "string",
+              "name": "domainLevel"
+            },
+            {
+              "type": "string",
+              "name": "domainScore"
+            },
+            {
+              "name": "criteriaLevelReport",
+              "type": "boolean"
+            },
+            {
+              "type": "string",
+              "name": "user_schoolName"
+            },
+            {
+              "type": "string",
+              "name": "user_schoolId"
+            },
+            {
+              "type": "string",
+              "name": "user_schoolUDISE_code"
+            },
+            {
+              "type": "string",
+              "name": "solution_type"
+            },
+            {
+              "type": "string",
+              "name": "organisation_name"
+            },
+            {
+              "type": "string",
+              "name": "user_boardName"
             }
-         }
-      },
-      "metricsSpec":[
-         {
-            "type":"floatSum",
-            "name":"question_response_number",
-            "fieldName":"questionResponseLabel_number"
-         }
-      ],
-      "granularitySpec":{
-         "type":"uniform",
-         "segmentGranularity":"day",
-         "queryGranularity":"day",
-         "rollup": true
+          ]
+        }
+      }
+    },
+    "metricsSpec": [
+      {
+        "type": "floatSum",
+        "name": "question_response_number",
+        "fieldName": "questionResponseLabel_number"
       }
-   },
-   "tuningConfig":{
-      "type":"kafka",
-      "reportParseExceptions":false
-      "maxRowsPerSegment": 5000000
-   },
-   "ioConfig":{
-      "topic":"{{env}}.sl.observation.druid",
-      "consumerProperties":{
-         "bootstrap.servers":"{{kafka_brokers}}"
-      },
-      "taskCount":"{{sl_observation_task_count}}",
-      "replicas":1,
-      "taskDuration":"PT14400S",
-      "useEarliestOffset": true,
-      "completionTimeout": "PT1800S"
-   }
+    ],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "DAY",
+      "queryGranularity": "none",
+      "rollup": false
+    }
+  },
+  "tuningConfig": {
+    "type": "kafka",
+    "reportParseExceptions": false
+  },
+  "ioConfig": {
+    "topic": "{{env}}.ml.observation.druid",
+    "consumerProperties": {
+      "bootstrap.servers": "{{kafka_brokers}}"
+    },
+    "taskCount": "{{ml_observation_task_count}}",
```

Comment:
Add these task count variables in defaults/main.yml. Same for all ingestion specs

Reply:
Added . Please check now